packages/react/src/ReactRenderer.tsx

```diff
@@ -121,7 +122,15 @@ export class ReactRenderer<R = unknown, P = unknown> {
       })
     }
 
-    this.render()
+    if (this.editor.isInitialized) {
```

Comment:
Wouldn't this call flushSync every time `setRenderer` was called after the editor has initialized?

Reply:
This is in the constructor function, for the initial render.

Before this change, we were calling flushSync on every setRenderer call (with `maybeFlushSync`).

With this change, it will only flush sync on the first render and then every update after will be handled with a `useSyncExternalStore` (i.e. letting React "pick up" the changes whenever it feels like it)